azurerm/resource_arm_mariadb_server.go

```diff
@@ -153,6 +153,15 @@ func resourceArmMariaDbServer() *schema.Resource {
 								string(mariadb.Disabled),
 							}, false),
 						},
+
+						"storage_autogrow": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ValidateFunc: validation.StringInSlice([]string{
```

Comment:
in the PostgreSQL version #4220 this is defaulted to 'Enabled' - do we want to do the same here?